client/lib/plugins/wporg-data/curated.json

```diff
@@ -0,0 +1,72 @@
+[
+	{
+		"name": "WooCommerce",
+		"author": "<a href=\"https://woocommerce.com\">Automattic</a>",
+		"rating": 92,
```

Comment:
Another thought that comes to mind is that perhaps we can give this a slightly different visual treatment, and omit the rating data.

cc @drw158